### PR TITLE
FFmpeg targets: allow to render alpha channel for webm and hap videos

### PR DESCRIPTION
**Pull request by rodolforg**
https://forums.synfig.org/t/any-good-way-to-export-a-single-video-with-alpha-transparency/12978
https://forums.synfig.org/t/transparent-background-for-videos/11381
https://github.com/synfig/synfig/issues/2161
https://github.com/synfig/synfig/issues/519

maybe:
https://github.com/synfig/synfig/issues/841

### CODE REVIEW

**Comment by rodolforg** [comment]
How to do it:
https://youtu.be/yVns8o493zI

It could be also codec `hap` with `.mov` extension

File: synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp [hunk 1]
```diff
@@ -244,7 +252,7 @@ ffmpeg_trgt::init(ProgressCallback *cb=NULL)
 	vargs.emplace_back("-f");
 	vargs.emplace_back("image2pipe");
 	vargs.emplace_back("-vcodec");
-	vargs.emplace_back("ppm");
+	vargs.emplace_back("pam");
```
Comment (ice0) [inline synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp:255]: Should it always be `pam`?

Comment (rodolforg) [reply synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp]: Whoops. Fixed.

**Review by ice0 — APPROVED** [review]

**Comment by rodolforg** [comment]
https://forums.synfig.org/t/alphachannel-videos-export-and-ffmpeg-profiles-obs-stinger-compatible/4405

**Pull request merged**